Add auto starting and archetype fields to structure

SayanC4 opened the pull request:
Add AutoStart (auto starting position) and RobotArchetype qualitative dropdown fields to structure.json

SayanC4 committed:
Add auto starting and archetype to structure.json

Add AutoStart (auto starting position) and RobotArchetype qualitative dropdown fields to structure.json

AnonymousCoder7777 reviewed (changes requested):
Looks great but I think the only options needed for start positions are front of the subwoofer, amp side, and source side

SayanC4 committed:
Amend structure.json with suggested AutoStart changes

AutoStart options are now "Subwoofer Front," "Amp Side," and "Source Side," as per Ryan's suggestion.
